linkml_runtime/utils/schemaview.py

```diff
@@ -342,6 +370,32 @@ def _order_rank(self, elements: dict):
         rank_ordered_elements.update(unranked_map)
         return rank_ordered_elements
 
+    def _order_inheritance(self, elements: DefDict) -> DefDict:
+        """
+        sort classes such that if C is a child of P then C appears after P in the list
```

Comment:
Quick nit: it's not just classes, inheritance can be applied to slots and subsets as well, so this is for _elements_ generally?

Reply:
Totally - I was just copying the existing comment there. Specifically the type bound is for Definitions since you need the `is_a` property